schemas/container.json

```diff
@@ -251,13 +278,19 @@
         "environment": {
           "type": "string",
           "minLength": 1
+        },
+        "environmentId": {
```

Comment:
Why do we need the property id? Where do we use it? If possible I think we should have an environment object object instead of 2 properties.